backend/Models/Request/UpdateUserRequest.cs

```diff
@@ -0,0 +1,9 @@
+namespace WhaleSpotting.Models.Request;
+
+public class UpdateUserRequest
+{
+    public int Id { get; set; }
```

Comment:
If we update the endpoint url per my suggestion above then we won't need the ID in this model anymore